priv/apps/itest/lib/transactions/encoding.ex

```diff
@@ -1,13 +1,13 @@
 defmodule Itest.Transactions.Encoding do
   @moduledoc """
-    Provides helper functions for encoding and decoding data.
+  Provides helper functions for encoding and decoding data.
   """
-  @output_type_marker <<1>>
+  @payment_tx_type_marker <<1>>
 
   alias Itest.Transactions.Deposit
 
   def get_data_for_rlp(%Deposit{inputs: inputs, outputs: outputs, metadata: metadata}),
-    do: [@output_type_marker, inputs, outputs, metadata]
+    do: [@payment_tx_type_marker, inputs, outputs, "", metadata]
```

Comment:
would it be more clearer to have this as a `0` instead according to the new format? https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/docs/integration-docs/integration-doc.md#payment-transaction-rlp-encoding

Reply:
argh, was going too fast here, 0 absolutely is better. I'll see if I haven't done a similar slip-up elsewhere